SDKBuildScripts/xplat_BuildAndTestAndroid.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+pushd ../../sdks/XPlatCppSdk/build/Android
+AndroidProjectPath=$PWD
+popd
+
+apkPath=$AndroidProjectPath/app/build/outputs/apk/debug/app-debug.apk
+testAssemblyDir="$1"
+
+CopyTestTitleData() {
+    cp -f "$PF_TEST_TITLE_DATA_JSON" "$AndroidProjectPath/app/src/main/assets"
+}
+
+BuildAPK() {
+    pushd "$AndroidProjectPath"
+    ./gradlew assembleDebug
+    popd
+}
+
+TestAPK() {
+    appcenter test run uitest --app "PlayFabSDKTeam/PlayFabXPlatAndroid" \
+    --devices ce2f4064 \
```

Comment:
I feel like this should be an environment variable, either in the job itself, or in the Random Env Vars setup script (depending on how often it may change).
Not this commit, but just something to note.